realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -169,6 +169,35 @@ private boolean containsDot(String s) {
         }
     }
 
+    /**
+     * Test if a field is null. Only works for relationships and RealmLists.
+     *
+     * @param fieldName - the field name
+     * @return The query object
+     * @throws java.lang.IllegalArgumentException if field is not a RealmObject or RealmList
+     */
+    public RealmQuery<E> isNull(String fieldName) {
+        // Currently we only support querying top-level
+        if (containsDot(fieldName)) {
+            throw new IllegalArgumentException("Following multiple relationships is not supported.");
```

Comment:
"Following multiple relationships" reads a bit unclear to me, Perhaps reword it to "Checking for null in nested objects is not supported."